RESTserver/src/main/java/com/softserve/if072/restservice/controller/ExceptionHandlerController.java

```diff
@@ -23,29 +25,39 @@
     @ResponseStatus(HttpStatus.NOT_FOUND)
     public String dataNotFound(DataNotFoundException e) {
         LOGGER.error(e.getMessage(), e);
+
         return null;
     }
 
     @ExceptionHandler(DataAccessException.class)
     @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
     public String dataAccessException(DataAccessException e) {
         LOGGER.error(e.getMessage(), e);
+
         return null;
     }
 
-
     @ExceptionHandler(AccessDeniedException.class)
     @ResponseStatus(HttpStatus.FORBIDDEN)
     public String accessDeniedException(AccessDeniedException e) {
         LOGGER.error(e.getMessage(), e);
+
         return null;
     }
 
-
     @ExceptionHandler(IllegalArgumentException.class)
     @ResponseStatus(HttpStatus.EXPECTATION_FAILED)
     public String IllegalArgumentException(IllegalArgumentException e) {
         LOGGER.error(e.getMessage(), e);
+
         return null;
     }
+
+    @ExceptionHandler(NotEnoughDataException.class)
+    @ResponseStatus(HttpStatus.UNPROCESSABLE_ENTITY)
+    public @ResponseBody String notEnoughDateException(NotEnoughDataException e) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)